docs/conf.py

```diff
@@ -16,12 +16,8 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-import sys, os
 import sphinx_bootstrap_theme
 
-
-sys.path.insert(0, os.path.abspath("../"))
```

Comment:
This is not needed? It seems to still be needed in [`spaghetti`](https://github.com/pysal/spaghetti/blob/fd8f98e0dd39f11360aab7c7bcb8e3b39da2f9c7/docs/conf.py#L17).

Reply:
im not sure about this. I've had to remove it from tobler and segregation otherwise the docs build will fail

Reply:
hmmm OK.